### ISSUE TITLE
Passing uniforms to shaders

### ISSUE THREAD

**Issue by RPrudden** [jovyan/pythreejs#75]
I am trying to use some custom shaders, which rely on a uniform 'dimensions' attribute.

<img width="891" alt="screen shot 2017-01-23 at 1 52 53 pm" src="https://cloud.githubusercontent.com/assets/9357195/22206463/4d0a2168-e173-11e6-809f-a2e80931ba20.png">

I can't find any way to send a value for dimensions to the shader. In thee.js, it would work something like: 
```
var material = new THREE.ShaderMaterial( {

	uniforms: {
		time: { value: 1.0 },
		resolution: { value: new THREE.Vector2() }
	},
	attributes: {
		vertexOpacity: { value: [] }
	},
	vertexShader: document.getElementById( 'vertexShader' ).textContent,
	fragmentShader: document.getElementById( 'fragmentShader' ).textContent

} );
```
However, there seems to be no equivalent to this in pythreejs. Has this not been implemented, or is there something I've missed?

**Comment by vidartf** [comment]
This has been fixed in master via #88, but has not really been tested (it might e.g. be necessary to set various `needsUpdate` flags). I'll leave this open until someone can confirm that it works as intended.

**Comment by vidartf** [comment]
Fixes and workable example in #125.
